cirq-google/cirq_google/api/v2/sweeps.py

```diff
@@ -91,6 +96,16 @@ def sweep_to_proto(
         for s in sweep.sweeps:
             sweep_to_proto(s, out=out.sweep_function.sweeps.add())
     elif isinstance(sweep, cirq.Linspace) and not isinstance(sweep.key, sympy.Expr):
+        if func:
+            try:
+                copied_linspace: cirq.Sweep = func(copy.deepcopy(sweep))
+                sweep = cast(cirq.Linspace, copied_linspace)
+            except Exception as e:
+                print(
+                    f"The function {func} was not applied to {sweep}."
+                    f" because there was an exception thrown: {str(e)}."
+                )
```

Comment:
Please delete this except-block, it may mask real errors in the `func`, and it makes it more tricky to check the exception trace in a debugger.  If `func` should leave some linspace sweeps unchanged, it should be up to the `func` to handle.

Reply:
+1 just call the method and use its result

Reply:
Good point! Updated.